src/createNumbersGenerator.js

```diff
@@ -25,8 +25,18 @@
  *
  * @return {number}
  */
-function createNumbersGenerator(numbers, min, max) {
-  // write code here
+function createNumbersGenerator(numbers = [], min = -Infinity, max = Infinity) {
+  const filteredNum = numbers.filter((item) => item >= min && item <= max);
+
+  return function() {
+    if (!filteredNum) {
+      return undefined;
+    } else {
+      const currentNum = filteredNum.splice(0, 1);
+
+      return currentNum[0];
```

Comment:
you can write simply 
```suggestion
      return filteredNum ? filteredNum.splice(0, 1) : undefined;
```

Reply:
or something like this